README.md

```diff
@@ -128,6 +126,13 @@ Set this variable to deploy partner pods in a custom namespace instead of the de
 export TNF_PARTNER_NAMESPACE="CNF-ns"
 ```
 
+### Specify the image id to be used with oc debug commands
+In disconnected environment, only specific versions of images are mirrored to the local repo. For the `oc debug` command (used by a number of tests) to work, set TNF_OC_DEBUG_IMAGE_ID:
```

Comment:
of images are mirrored -> [of images that are mirrored | of mirrored image]

Reply:
huh?

Reply:
right right!